### ISSUE TITLE
when I change the orientation from portrait to landscape and back to portrait, slide always crashes

### ISSUE THREAD

**Issue by Fernando-UofT** [ccrama/Slide#235]
## Info
In android 5.1.1, open Slide in portrait orientation. Change orientation to landscape, and go back to portrait. My device is nexus 6
## Stacktrace
```
java.lang.ClassCastException: me.ccrama.redditslide.Views.PreCachingLayoutManager cannot be cast to android.support.v7.widget.StaggeredGridLayoutManager
at me.ccrama.redditslide.Fragments.SubmissionsView.onConfigurationChanged(SubmissionsView.java:51)
at android.support.v4.app.Fragment.performConfigurationChanged(Fragment.java:2020)
at android.support.v4.app.FragmentManagerImpl.dispatchConfigurationChanged(FragmentManager.java:2101)
at android.support.v4.app.FragmentController.dispatchConfigurationChanged(FragmentController.java:246)
at android.support.v4.app.FragmentActivity.onConfigurationChanged(FragmentActivity.java:267)
at android.support.v7.app.AppCompatActivity.onConfigurationChanged(AppCompatActivity.java:129)
at android.app.ActivityThread.performConfigurationChanged(ActivityThread.java:4044)
at android.app.ActivityThread.handleConfigurationChanged(ActivityThread.java:4117)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1413)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5254)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
```
Slide v4.2.1, API v22
Reported via Slide Crash Capture

**Comment by markshawtoronto** [comment]
This happens on my device too:

Nexus 4 - Android 5.1.1.

**Comment by ccrama** [comment]
Fixed with f369367292f69357f3eaced1b9af4ff472438cd1!

**Comment by markshawtoronto** [comment]
Yay :-)
